magpie/api/management/register/register_views.py

```diff
@@ -82,3 +91,21 @@ def leave_discoverable_group_view(request):
     group = ru.get_discoverable_group_by_name(group.group_name, db_session=request.db)
     uu.delete_user_group(user, group, request.db)
     return ax.valid_http(http_success=HTTPOk, detail=s.RegisterGroup_DELETE_OkResponseSchema.description)
+
+
+@s.TemporaryUrlAPI.get(schema=s.TemporaryURL_GET_RequestSchema, tags=[s.RegisterTag],
+                       response_schemas=s.TemporaryURL_GET_responses)
+@view_config(route_name=s.TemporaryUrlAPI.name, request_method="GET", permission=NO_PERMISSION_REQUIRED)
+def handle_temporary_url(request):
+    """
+    Handles the operation according to the provided temporary URL token.
+    """
+    token = ar.get_value_matchdict_checked(request, key="token", pattern=ax.UUID_REGEX)
+    token = token.split(":")[-1]  # remove optional prefix if any
+    db_session = request.db
+    tmp_url = models.TemporaryToken.by_token(token, db_session=db_session)
```

Comment:
by_token and handle_temporary_token don't seem to deal with url but rather with an actual TemporaryToken. Variable name is misleading

Reply:
Ah, yes, remnant of previous code. Will adjust.